src/relay/pass/let_list.h

```diff
@@ -0,0 +1,75 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file let_list.h
+ * \brief Let List Let you Let stuff yo.
+ */
+#ifndef TVM_RELAY_PASS_LET_LIST_H_
+#define TVM_RELAY_PASS_LET_LIST_H_
+
+#include <utility>
+#include <vector>
+#include <tuple>
+#include "tvm/relay/type.h"
+#include "incomplete_type.h"
+
+namespace tvm {
+namespace relay {
+
+struct PartialVar {
```

Comment:
for now, move let_list data structure into dead_code_elim since it is only used in one place, we can get it as a header latter

Reply:
done